src/tiler/styles.mss

```diff
@@ -36,6 +36,13 @@
 }
 
 @streamColor: #1562A9;
+@wtrQualColor0: #d7191c;
+@wtrQualColor1: #fdae61;
+@wtrQualColor2: #ffffbf;
+@wtrQualColor3: #a6d96a;
+@wtrQualColor4: #1a9641;
+@wtrQualColorNA: #648db9;
```

Comment:
The blue for `NA` here seems a little too in-line with the symbology for actual data and close to the regular stream color.  I'd suggest a dark gray or something to indicate more clearly that there is no water quality data.